iso8859-1/src/lib.rs

```diff
@@ -344,7 +344,9 @@ mod api_tests {
         for byte in b'0'..=b'9' {
             assert!(IsoLatin1Char(byte).is_numeric());
         }
-        todo!()
+        for byte in b'a'..=b'z' {
+            assert!(!IsoLatin1Char(byte).is_numeric());
+        }        
```

Comment:
That is very good, but also don't forget to add tests for the second part of the character table as well

Reply:
@GrayJack good point! I will create an array with all numeric values and then I will test those in the for-loop.